mlflow/catboost.py

```diff
@@ -39,17 +39,18 @@
 _MODEL_BINARY_FILE_NAME = "model.cb"
 
 
+def _get_default_pip_requirements():
```

Comment:
shall we make this a public function that users can reference? (and make a similar change for other flavors)

Reply:
sure!